### PR TITLE
Updated App protection Global policy section

### PR DESCRIPTION
**Pull request by balktl**
I added a note with a link to other documentation stating that the OneDrive Admin Center is no longer being updated and that Microsoft Endpoint Manager can be used.

Addresses issue #1553

### CODE REVIEW

**Comment by PRMerger19** [comment]
@balktl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Erikre — APPROVED** [review]

**Comment by PRMerger8** [comment]
@Erikre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Erikre** [comment]
@balktl - Thanks for the contribution. #sign-off

File: memdocs/intune/apps/app-protection-policy.md [hunk 1]
```diff
@@ -208,6 +208,9 @@ By default, there can only be one **Global** policy per tenant. However, you can
 
 While the **Global** policy applies to all users in your tenant, any standard Intune app protection policy will override these settings.
 
+> [!NOTE]
+> The policy settings in the OneDrive Admin Center are no longer being updated. Microsoft Enpoint Manager may be used instead. For more information, see [Control access to features in the OneDrive and SharePoint mobile apps](/onedrive/control-access-to-mobile-app-features.md).
```
Comment (ShannonLeavitt) [inline memdocs/intune/apps/app-protection-policy.md:212]:
```suggestion
> The policy settings in the OneDrive Admin Center are no longer being updated. Microsoft Enpoint Manager may be used instead. For more information, see [Control access to features in the OneDrive and SharePoint mobile apps](/onedrive/control-access-to-mobile-app-features).
```
trying to fix link

**Comment by PRMerger18** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/apps/app-protection-policy.md [hunk 2]
```diff
@@ -208,6 +208,9 @@ By default, there can only be one **Global** policy per tenant. However, you can
 
 While the **Global** policy applies to all users in your tenant, any standard Intune app protection policy will override these settings.
 
+> [!NOTE]
+> The policy settings in the OneDrive Admin Center are no longer being updated. Microsoft Enpoint Manager may be used instead. For more information, see [Control access to features in the OneDrive and SharePoint mobile apps](/onedrive/control-access-to-mobile-app-features).
```
Comment (ShannonLeavitt) [inline memdocs/intune/apps/app-protection-policy.md:212]:
```suggestion
> The policy settings in the OneDrive Admin Center are no longer being updated. Microsoft Enpoint Manager may be used instead. For more information, see [Control access to features in the OneDrive and SharePoint mobile apps](/onedrive/control-access-to-mobile-app-features.md).
```
Reverting previous change

**Comment by PRMerger6** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ShannonLeavitt** [comment]
@Erikre - The link didn't work for me, so I tried a fix, which still didn't work. Could you circle back after the publish to live to verify that the link works? Merging now.

**Pull request merged**